zap/src/parser/grammar.lalrpop

```diff
@@ -103,6 +103,13 @@ TyKind: SyntaxTyKind<'input> = {
 
     "string" <r:("(" <IntRange> ")")?> => SyntaxTyKind::Str(r),
     "buffer" <r:("(" <IntRange> ")")?> => SyntaxTyKind::Buf(r),
+    "vector" "(" ")" => SyntaxTyKind::Vector(
```

Comment:
For parity with other types such as `Instance` vs `Instance (BasePart)`, let's change it to:
```suggestion
    "vector" => SyntaxTyKind::Vector(
```